install/install.sh

```diff
@@ -95,7 +95,7 @@ fi
 server_ip=`dig +short myip.opendns.com @resolver1.opendns.com`
 read -e -p "Server IP: " -i "${server_ip}" config_server_ip
 read -e -p "Server interface (SSL): " -i "443" config_server_port
-read -e -p "Application root (if not a whole (sub)domain, enter the path. None if whole (sub)domain): " -i "None" config_application_root
+read -e -p "Application root (if not a whole (sub)domain, enter the path. None if whole (sub)domain): " -i "'/'" config_application_root
```

Comment:
I thought (at least under the Flask version I used) that it should be `None`, as we'd use a full domain (e.g. `canihavesome.coffee`) and not a subdirectory... Did it change?

Reply:
The web page continues to jump back to `auth/login` if the `application_root` is `None`. Before I find any other supportive material, let's leave that to `None`.